astroid/tests/unittest_inference.py

```diff
@@ -3309,6 +3309,8 @@ class Entity(object):
         self.assertIsInstance(attributes[0], nodes.Const)
         self.assertEqual(attributes[0].value, 1)
 
+
+    @pytest.mark.xfail(reason="These call should be uninferable because they are invalid calls")
```

Comment:
Do we still expect the test to fail even if we modified the test to check for an exception instead?

Reply:
It's more of a `TODO` thing that I can fix in another issue where these tests should be raising errors because they are invalid `type.__new__` calls. The previous asserts were testing for incorrect behavior